src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$cell-color:#eee;
+$cell-size: 100px;
+$cell-gap: 1px;
+$calendar-padding: 10px;
+$calendar-width: ($cell-size + $calendar-padding + $cell-gap) * 7;
```

Comment:
<img width="855" alt="Screenshot 2023-05-01 at 19 55 06" src="https://user-images.githubusercontent.com/117587463/235492124-fe2f9fe2-b470-4b59-8aeb-c159bfdb3332.png">

The formula for calculating the calendar width seems to be incorrect. It should be calculated as ($cell-size * 7) + ($cell-gap * 6) + ($calendar-padding * 2).